examples/README.md

```diff
@@ -320,6 +320,7 @@ Example | Description
 [Observers](../examples/ecs/observers.rs) | Demonstrates observers that react to events (both built-in life-cycle events and custom events)
 [One Shot Systems](../examples/ecs/one_shot_systems.rs) | Shows how to flexibly run systems without scheduling them
 [Parallel Query](../examples/ecs/parallel_query.rs) | Illustrates parallel queries with `ParallelIterator`
+[Prefabs](../examples/ecs/prefabs.rs) | Demonstrates how to load, store and clone template entity collections
```

Comment:
I think this deserves a qualifier or a different more specific name less loaded with less context. People will see this name and think "this is the Bevy prefab system", which has implications like "an asset format", "inheritance", etc.

Reply:
Good call; I definitely think that's the right call for the current stage of maturity.